practices/secrets.md

```diff
@@ -56,6 +56,10 @@ To use `sops` with a 2i2c configuration file, follow these steps:
 3. **Set up the Google Cloud Key Management Service (KMS)**. This allows you to use your Google Cloud login to provide authentication for `sops`.
    [Follow the `sops` instructions to use KMS](https://github.com/mozilla/sops/#encrypting-using-gcp-kms).
 
+```{note}
+Step 3 is only required when setting up `sops` for the first time.
+```
```

Comment:
Felt good to avoid a reference to a specific step number. Note that markdown allow you to write `1.` repeatedly as well and render into 1, 2, 3 ...

Having seen documentation get outdated I trust a direct/nested reference referencing "this step" more as a reader as well even though it was maintained correctly.

````suggestion
   ```{note}
   This step is only required when setting up `sops` for the first time.
   ```
````